server/controllers/products.js

```diff
@@ -11,7 +12,33 @@ module.exports = {
     })
       .then(({ data }) => res.json(data))
       .catch((err) => {
-        console.error('There was a problem in the server retrieving product data: ', err);
+        console.log('There was a problem in the server retrieving product data: ', err);
+        res.sendStatus(404);
+      });
+  },
+
+  getSpecificProduct(req, res) {
+    axios.get(`${ATELIER_API}/products/${req.params.product_id}`, {
+      headers: {
+        authorization: API_TOKEN,
+      },
+    })
+      .then(({ data }) => res.json(data))
+      .catch((err) => {
+        console.log('There was a problem in the server retrieving specific product data: ', err);
+        res.sendStatus(404);
+      });
+  },
+
+  getProductStyles(req, res) {
+    axios.get(`${ATELIER_API}/products/${req.params.product_id}/styles`, {
+      headers: {
+        authorization: API_TOKEN,
+      },
+    })
+      .then(({ data }) => res.json(data))
+      .catch((err) => {
+        console.log('There was a problem in the server retrieving product styles: ', err);
         res.sendStatus(404);
       });
   },
```

Comment:
it looks like we both created a getStyles method. Looks almost exactly the same as mine so I guess it's up to you whether you want to keep this one or use mine